### PR TITLE
Change create-vote count param to int64

### PR DESCRIPTION
**Pull request by samarthtambad**

### CODE REVIEW

File: vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const MsgCreateVote = {
       writer.uint32(18).string(message.receiver);
     }
     if (message.count !== 0) {
-      writer.uint32(24).int32(message.count);
+      writer.uint32(24).int64(message.count);
```
Comment (prtk418) [inline vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts:32]: Tested the changes on UI too?

File: vue/src/store/generated/mande-chain/mandechain.voting/module/types/cosmos/base/query/v1beta1/pagination.ts [hunk 2]
```diff
@@ -38,12 +38,6 @@ export interface PageRequest {
    * is set.
    */
   count_total: boolean;
-  /**
-   * reverse is set to true if results are to be returned in the descending order.
-   *
-   * Since: cosmos-sdk 0.43
-   */
-  reverse: boolean;
```
Comment (prtk418) [inline vue/src/store/generated/mande-chain/mandechain.voting/module/types/cosmos/base/query/v1beta1/pagination.ts:46]: I know these are generated changes but not sure if we should push these
We might need to test the UI properly so that none of the APIs break with these changes

File: vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts (hunk 1 above)
Comment (samarthtambad) [reply vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts:32]: Yes. I tested the changes both on UI and cli. It is working fine.

On UI we have a limit for 1000 so could only test with that value.

File: vue/src/store/generated/mande-chain/mandechain.voting/module/types/cosmos/base/query/v1beta1/pagination.ts (hunk 2 above)
Comment (samarthtambad) [reply vue/src/store/generated/mande-chain/mandechain.voting/module/types/cosmos/base/query/v1beta1/pagination.ts]: Ideally, the generated changes should not break the UI. 
Also, I've tested it to not break existing functionality.

File: vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts (hunk 1 above)
Comment (prtk418) [reply vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts:32]: Can you please remove the 1000 limit condition from the code and then test. Just to be sure
For ref - https://github.com/mande-labs/mande-chain/pull/23/files#diff-3b8055e901d876b3fdae9172632d08a64592a9c9474e8ef312fe6445cebae3aaR291

Comment (samarthtambad) [reply vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts:32]: Tested that. The first TX for 5000mand goes through but the second one gets internal server error.

Comment (samarthtambad) [reply vue/src/store/generated/mande-chain/mandechain.voting/module/types/voting/tx.ts:32]: I think it's better if I don't include UI changes in the PR. Better to do it separately.

File: vue/src/store/generated/mande-chain/mandechain.voting/module/types/cosmos/base/query/v1beta1/pagination.ts (hunk 2 above)
Comment (samarthtambad) [reply vue/src/store/generated/mande-chain/mandechain.voting/module/types/cosmos/base/query/v1beta1/pagination.ts]: Will close this PR and create a new one without including UI changes.
